pkg/e2etemplates/aws_operator_chart_values.go

```diff
@@ -20,37 +22,127 @@ const AWSOperatorChartValues = `Installation:
       SSH:
```

Comment:
In the template basically all env var placeholders are replaced with template placeholders.